webpack/configs/base.ts

```diff
@@ -0,0 +1,36 @@
+import * as webpack from "webpack";
+
+import { TsConfigPathsPlugin } from "awesome-typescript-loader";
+
+const CommongConfig: webpack.Configuration = {
+
+    entry: "./webpack/globalize.ts",
+
+    output: {
+        path: `${__dirname}/../../build`
+    },
+
+    resolve: {
+        extensions: [".ts", ".mjs", ".js", ".json"],
+        plugins: [new TsConfigPathsPlugin()]
+    },
+
+    module: {
+        rules: [
+            // All files with a '.ts' extension will be handled by 'awesome-typescript-loader'.
+            { test: /\.ts$/, loader: "awesome-typescript-loader" },
+
+            { test: /\.mjs$/, include: /node_modules/, type: "javascript/auto" },
+
+            // All output '.js' files will have any sourcemaps re-processed by 'source-map-loader'.
+            { enforce: "pre", test: /\.js$/, loader: "source-map-loader" }
+        ]
+    },
+
+    performance: {
+        hints: false
```

Comment:
What does this mean? If you can add comment to explain the webpack config entries that would be great.

Reply:
Performance/Bundle size notifications from webpack. I should actually enable it as warning for development.

---
In reply to: [250238985](https://github.com/Microsoft/clarity-js/pull/126#discussion_r250238985) [](ancestors = 250238985)

Reply:
I don't think it's worth explaining every single property,because there is a Webpack Configuration documentation page out there. What i did is I placed a link to those docs at the top of each file.

---
In reply to: [250336418](https://github.com/Microsoft/clarity-js/pull/126#discussion_r250336418) [](ancestors = 250336418,250238985)